components/list/list-controls.js

```diff
@@ -52,6 +52,10 @@ export class ListControls extends SelectionControls {
 		if (this._extendSeparator) this.style.setProperty('--d2l-selection-controls-padding', '0px');
 	}
 
+	updateParentHasAddButon() {
+		// N/A
```

Comment:
Since `list-controls` is a subscriber, it's affected by the `list` change below to use the subscriber code